src/AspNet.Security.OAuth.Docusign/DocusignAuthenticationConstants.cs

```diff
@@ -0,0 +1,22 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+namespace AspNet.Security.OAuth.Docusign;
+
+public static class DocusignAuthenticationConstants
```

Comment:
I would put this with the defaults, and also use readonly fields like we do everywhere else for the URLs. We've had providers change things in the past which can make constants problematic for endpoint URLs.